acceptance/services_test.go

```diff
@@ -147,7 +147,7 @@ var _ = Describe("Services", func() {
 			Expect(out).To(
 				HaveATable(
 					WithHeaders("NAME", "CREATED", "CATALOG SERVICE", "STATUS", "APPLICATIONS"),
-					WithRow(service, WithDate(), catalogService.Meta.Name, "not-ready", ""),
+					WithRow(service, WithDate(), catalogService.Meta.Name, "not-ready|deployed", ""),
```

Comment:
Please check the resulting regex. We may need parens around the alternation to get something sensible, i.e. `(not-ready|deployed)`, to separate it from the surrounding elements (Which check for a sequence of spaces, using `*`).

Reply:
Good point, fixed here: https://github.com/epinio/epinio/pull/1521/commits/9d104ede0469453cbd61a5b155e5efe01791995e

Reply:
Thanks.